code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -439,6 +439,36 @@ datum
 			on_mob_life(var/mob/M, var/mult = 1)
 				if(!M) M = holder.my_atom
 				M.stuttering += rand(0,2)
+				if(M.client && probmult(5))
+					for (var/obj/critter/domestic_bee/bee in range(7,M))
+						var/chat_text = null
+						var/shit_bees_say = list("Hello [M].",
```

Comment:
I'd prefer if this was in a txt file but since you're using `[]` you'd need to use the mildly janky smart_string_picker.